core-web/apps/dotcms-ui/src/app/api/services/dot-format-date-service.ts

```diff
@@ -104,9 +104,10 @@ export class DotFormatDateService {
     }
```

Comment:
<img width="2435" alt="image" src="https://user-images.githubusercontent.com/751424/219146837-9c9fb8ad-333d-4b51-95a7-bf68f94e1031.png">

Check the design.

Also the empty loading should not have animation because it gave the impression that there is something loading and is not.

Reply:
<img width="671" alt="image" src="https://user-images.githubusercontent.com/751424/219147138-19851274-2265-418b-bf67-e7641d269c1b.png">

This gray should be lighter, check the vars if we're missing let's figure it something out.

Reply:
<img width="276" alt="image" src="https://user-images.githubusercontent.com/751424/219147736-00bf91f1-71c7-48b9-aa0c-71b402da0293.png">

Group this 3 in the same cell so when is a really big screen it doesn't separate that much.

Reply:
<img width="401" alt="image" src="https://user-images.githubusercontent.com/751424/219149953-f1513405-ff9a-44e4-98e0-a484c9c894bb.png">

The Status label should be aligned to the left.

Reply:
<img width="240" alt="image" src="https://user-images.githubusercontent.com/751424/219154673-ad8ac81a-b6f5-445a-af61-4f1cf8d4314c.png">

I think we should have "Add / Remove" to favorite here... it doesn't make sense that I have to go to edit page to edit a page top fav when I am in the fav page.

Reply:
<img width="1812" alt="image" src="https://user-images.githubusercontent.com/751424/219155046-3bb5e795-9d49-4918-998a-d091f8205ac3.png">

We need an empty state for the table... Melissa has been informed.

Reply:
<img width="1294" alt="image" src="https://user-images.githubusercontent.com/751424/219155407-0ad3c2dc-0e55-4b2a-a42d-cae8dffbbd87.png">

Missing this shadow.

Reply:
<img width="1629" alt="image" src="https://user-images.githubusercontent.com/751424/219156438-40c7f47f-0c46-4f26-8e62-32ad96c064ad.png">

I think we should do the math and fill the space. Ask Will tho.

Reply:
fixed
![image](https://user-images.githubusercontent.com/37185433/219186854-d35b9b80-65af-4a82-9862-720e4a05c985.png)

![image](https://user-images.githubusercontent.com/37185433/219190127-e8d82c71-2797-41b4-800a-61185aded616.png)

pending:
- empty state for the table (design not ready)
- do the math and fill the space of favorite page cards (to discuss)

Reply:
Nice!!! I miss one, the three dots button should be always aligned right and have the margin right of the row

Reply:
fixed!